### ISSUE TITLE
Aborting bundle extraction leads to cache corruption

### ISSUE THREAD

**Issue by guillaumerose** [code-ready/crc#1903]
### General information

OS: All

## CRC version

All

### Steps to reproduce

```
$ crc setup
INFO Checking if CRC bundle is extracted in '$HOME/.crc' 
INFO Extracting bundle from the CRC executable    
crc.qcow2: 163.09 MiB / 9.85 GiB [-->______________________________________________________________________________________________________________________________________________________________________________________] 1.62%^C

$ crc setup
Setup is complete, you can now run 'crc start' to start the OpenShift cluster

$ du -h ~/.crc/cache/crc_libvirt_4.6.9
166M	/home/guillaumerose/.crc/cache/crc_libvirt_4.6.9

$ crc start -p ~/pull-secret.txt
...
INFO Checking if libvirt 'crc' network is active  
INFO Loading bundle: crc_libvirt_4.6.9.crcbundle ... 
INFO Verifying bundle crc_libvirt_4.6.9.crcbundle ... 
Invalid bundle crc_libvirt_4.6.9.crcbundle: oc not found in bundle

```

Then the VM cannot start correctly.

**Comment by cfergeau** [comment]
I happen to have been working on a PR which would address this among other things.

**Comment by cfergeau** [comment]
Fixed in 1.22
